ArWikiCats/legacy_bots/ma_bots2/lab.py

```
@@ -433,15 +304,41 @@ def get_con_lab(separator: str, country: str, start_get_country2: bool = False)
+        "for member-of-parliament": "لعضوية البرلمان",
+    }
 
-    label = _lookup_label_from_sources(lookup_chain, country_lower, log_context=f"get_con_lab({country_lower})")
+    lookup_chain = {
+        # NOTE: resolve_nat_genders_pattern_v2 IN TESTING HERE ONLY
+        # "resolve_nat_genders_pattern_v2" : lambda t: resolve_nat_genders_pattern_v2(t),
+        "all_new_resolvers": lambda t: all_new_resolvers(t),
+        "get_from_new_p17_final": lambda c: get_from_new_p17_final(c),
+        "pf_keys2": lambda c: get_from_pf_keys2(c),
+        "get_pop_All_18": lambda c: get_pop_All_18(c, ""),
+        "get_pop_All_18_no_dash": lambda c: get_pop_All_18(country_no_dash, ""),
```

Ruff (0.14.13)

Unused lambda argument: `c`

(ARG005)